mail_tracking_mailgun/tests/test_mailgun.py

```diff
@@ -405,7 +405,7 @@ def test_email_validity(self, mock_request):
         }
         with self.assertRaises(UserError):
             self.partner.check_email_validity()
-        self.assertTrue(self.partner.email_bounced)
+            self.assertTrue(self.partner.email_bounced)
```

Comment:
With this change, you don't execute this line, if the previous line raise an UserError the next line is not executed, I think this test is not valid in this way

Reply:
yes i just having testing and i forget to remove it. The reason is because i have problem with this test.
If you could give me some hint to solve this bug with this line of the test welcome. 
Thanks